### DISCUSSION TITLE
Domain decomposition method and Inverse PINN for coefficients that can vary within domain

### DISCUSSION THREAD

**Discussion by ryabs07** [lululxvi/deepxde#1701] [Q&A]
Hi,
Is it possible to implement domain decomposition methods within DeepXDE framework? 

And, is it possible to apply PINNs for inverse problems where the coefficient can vary within the domain?

Any leads would be highly appreciated :)
Thank you

**Comment by lululxvi** [comment]
Yes, possible. 

Read the demo examples for the 2nd question.

**Reply by ryabs07** [reply]
Thank you for the reply Dr. Lu. I found the demo for space-dependent inverse problem.
Are there also any demo for implementation of domain decomposition?
